lib/livebook/runtime/fly.ex

```diff
@@ -311,6 +311,20 @@ defmodule Livebook.Runtime.Fly do
     end
   end
 
+  defp default_flyctl_path() do
+    # Checks the default locations where flyctl gets installed using
+    # the official instructions
+
+    home = System.user_home()
+
+    paths = [
+      "/opt/homebrew/bin/flyctl",
```

Comment:
I'll check the exact location tomorrow but on x64 Macs, homebrew installs things to a different directory, /usr/local/Cellar and I think there are symlinks too. I'm just mentioning this here in case I forget to follow up. :)

Reply:
On my x64 Mac flyctl got installed to `/usr/local/bin/fly` which is a symlink to `/usr/local/Cellar/flyctl/0.2.96/bin/fly`. So yeah, we're good, sorry for the noise!

Reply:
Perfect, thanks for checking!